dask_ml/_partial.py

```diff
@@ -105,11 +105,13 @@ def _get_predict_dtype(self, X):
 def _partial_fit(model, x, y, kwargs=None):
     kwargs = kwargs or dict()
     start = tic()
-    logger.info("Starting partial-fit %s", dask.base.tokenize(model, x, y))
+    if logger.getEffectiveLevel() == logging.DEBUG:
+        token = dask.base.tokenize(model, x, y)
+        logger.debug("Starting partial-fit %s", token)
```

Comment:
We just shouldn't call tokenize here at all.  It's not strictly necessary.  I suspect that these logging lines were added during a debug session.  I think that we should remove them.

Reply:
More generally, I think that we probably shouldn't be logging at this fine of granularity.  A user looking through logs would not be happy learning that there were thousands of lines every time they called fit on a dask-ml model

Reply:
Ah, good point. I've removed them.